### PR TITLE
We're happy with the name "Global".

### PR DESCRIPTION
**Pull request by nlewycky**

### CODE REVIEW

**Review by jubianchi — APPROVED** [review]

File: lib/vm/src/global.rs [hunk 1]
```diff
@@ -6,10 +6,9 @@ use thiserror::Error;
 use wasmer_types::{GlobalType, Mutability, Type, Value};
 
 #[derive(Debug)]
-/// TODO: figure out a decent name for this thing
```
Comment (syrusakbary) [inline lib/vm/src/global.rs:9]:
```suggestion
/// A Global instance
```

Comment (nlewycky) [reply lib/vm/src/global.rs:9]: github doesn't let me commit suggestions that replace deleted lines. :(

**Pull request merged**